packages/eslint-plugin-office-addins/src/rules/no-empty-load.ts

```diff
@@ -0,0 +1,34 @@
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+
+export = {
+  name: "no-empty-load",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      emptyLoad:
+        "Calling an empty load can slow down your add-in",
+    },
+    docs: {
+      description:
+        "Calling an empty load can cause load unneeded data and slow down your add-in",
+      category: <
+        "Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors"
+      >"Best Practices",
+      recommended: <false | "error" | "warn">false,
+      url: "https://docs.microsoft.com/en-us/office/dev/add-ins/develop/application-specific-api-model?view=powerpoint-js-1.1#calling-load-without-parameters-not-recommended",
+    },
+    schema: [],
+  },
+  create: function (context: any) {
+    return {
+      "CallExpression[callee.property.name='load']:not([arguments>Literal])"(
```

Comment:
Without type information we can't say that the load call is actually coming from an office object . . . but in other rules we are searching for object that use get methods in order to determine if they are the variables we are looking for.

Reply:
Sure, I will try to make this check. Will take more time now as the code won't be just a query search.